client/logmon/logmon_test.go

```diff
@@ -16,13 +17,23 @@ import (
 
 func TestLogmon_Start_rotate(t *testing.T) {
 	require := require.New(t)
+
+	stdoutLog := "stdout"
+	stderrLog := "stderr"
```

Comment:
Do you happen to know the value of having these as variables or their significance?  If they aren't used anywhere else, I might suggest using literal in the `LogConfig` initialization directly.

Reply:
I have no idea. I assume they were used prior to some past refactoring somewhere else.